Engine/src/Engine/Entity.cpp

```diff
@@ -0,0 +1,8 @@
+#include "entity.h"
+
+namespace Engine
+{
+    Entity::Entity(entt::entity entity_handle, Scene* scene)
+        : m_entityHandle(entity_handle), m_scene(scene)
```

Comment:
use entityHandle instead of entity_handle for style guidelines
also maybe put each parameter assignment on its own line?

Reply:
done